### ISSUE TITLE
Mica effect completely broken

### ISSUE THREAD

**Issue by Magirus-Deutz** [MicaForEveryone/MicaForEveryone#157]
I installed MicaforEveryone, but why is mine looks like that?

![ss](https://user-images.githubusercontent.com/20948362/198039907-801af216-44e6-4efd-a5bd-46d26520bd05.png)

I tested this software on the modern laptop and I had a similar result as screenshots. The laptop isn't mine, so i cant reach the laptop again, and then i decided to test this software on the virtual machine. 

I tested on the vm, and the same problem with the laptop 

Why this happening? is video driver so old or unsupported? or is Windows versions which i used unsupported?

I didn't put the bug tag or something, because i think this is not a bug, i just wanna get answer this. (if its a bug, then we can put the bug tag or something later.)

Laptop has Windows 11 Build 22622.575
VM has Windows 11 Build 22621.1

![cool gradient](https://user-images.githubusercontent.com/20948362/198039934-15aff0e5-a726-46bc-9193-a7a6bb725dfc.png)

**Comment by dongle-the-gadget** [comment]
Did you enable Blur Behind or Extend Frame Into Client Area?

**Comment by Magirus-Deutz** [comment]
Yes, i enabled it, but most people enabling the blur behind for the explorer, why is mine broken?

**Comment by xmine64** [comment]
> most people enabling the blur behind for the explorer, why is mine broken?

Nobody does that. It won't work well on explorer.

**Comment by dongle-the-gadget** [comment]
As explained in the Wiki, Blur Behind and Extend Frame Into Client Area has the potential to cause UI problems, so if that's the case, disable these two options.

**Comment by Magirus-Deutz** [comment]
ok, i disabled the blur behind, but the extend frame option is still enabling and it's working correctly now, (almost).

But, how people make every Explorer window mica? Do I need special msstyle or another software for this?

**Comment by dongle-the-gadget** [comment]
Yeah you do.

**Comment by Magirus-Deutz** [comment]
ok then, thanks all for helping me.
